tests/model_serving/model_runtime/vllm/constant.py

```diff
@@ -0,0 +1,76 @@
+from typing import Any, Dict
+
+GRPC_PORT = 8033
+REST_PORT = 8080
+# Configurations
+vLLM_CONFIG: Dict[str, Dict[str, Any]] = {
+    "port_configurations": {
+        "grpc": [{"containerPort": GRPC_PORT, "name": "h2c", "protocol": "TCP"}],
+        "raw": [
+            {"containerPort": REST_PORT, "name": "http1", "protocol": "TCP"},
+            {"containerPort": GRPC_PORT, "name": "h2c", "protocol": "TCP"},
+        ],
+    },
+    "commands": {"GRPC": "vllm_tgis_adapter"},
+}
+
+TEMPLATE_MAP = {
+    "nvidia": "vllm-runtime-template",
```

Comment:
Maybe create a class `AcceleratorType` in constants.py, and reuse across the PR e.g. `AcceleratorType.NVIDIA`

Reply:
done